packages/react-renderer-demo/src/app/pages/renderer/deleted-value.md

```diff
@@ -0,0 +1,20 @@
+import Grid from '@material-ui/core/Grid'
+import RawComponent from '@docs/raw-component';
+
+import ListOfContents from '../../src/helpers/list-of-contents';
+
+<Grid container item>
+<Grid item xs={12} md={10}>
+
+# Deleted value
+
+If you required a specific value for a field which had **initialValue** to set it to some specific value. For instance if you have a form which edits some entity and you need to delete some attribute, APIs usually require the value to be set to `null` to register the change.
```

Comment:
Maybe it's just me, but the first sentence doesn't make any sense :confused: I can probably guess that the field will send some value if make it empty, which in our cases needs to be `null`. If that's the case, I'd name it `emptyValue`, `clearValue` or `clearedValue` and the description could be something like this:

> The value to send upon a submit if the field is empty. This wildcard value can be used to distinguish between an untouched field and a cleared one. For example if you have a form that edits an entity and you would like to clear an attribute. Some APIs require the value to be set to `null` to register the change.

What do you guys think?